tests/serverpod_test_client/lib/src/protocol/defaults/uuid/uuid_default.dart

```diff
@@ -0,0 +1,130 @@
+/* AUTOMATICALLY GENERATED CODE DO NOT MODIFY */
+/*   To generate run: "serverpod generate"    */
+
+// ignore_for_file: library_private_types_in_public_api
+// ignore_for_file: public_member_api_docs
+// ignore_for_file: implementation_imports
+// ignore_for_file: use_super_parameters
+// ignore_for_file: type_literal_in_constant_pattern
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'package:serverpod_client/serverpod_client.dart' as _i1;
+import 'package:uuid/uuid.dart' as _i2;
+
+abstract class UuidDefault implements _i1.SerializableModel {
+  UuidDefault._({
+    this.id,
+    _i1.UuidValue? uuidDefaultRandom,
+    _i1.UuidValue? uuidDefaultRandomNull,
```

Comment:
Question(For my knowledge): Why are both of these nullable in the model? As far as I understand, only one of them should be nullable.

Reply:
No, both should be nullable (optional) because they will have the default value on constructor initialization
![Screenshot 2024-08-15 at 9 13 33 AM](https://github.com/user-attachments/assets/d5fe4796-12cd-4f48-afcc-c0e9391a08e4)

but if you check the model field, only one of these is nullable:
![Screenshot 2024-08-15 at 9 14 16 AM](https://github.com/user-attachments/assets/0aed219a-16b2-4140-8051-b0830fd8bedd)